CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java

```diff
@@ -60,6 +59,9 @@
     private DoubleProperty currentRoll;
     private DoubleProperty leftDistanceToWall;
     
+    private double oldLeftDistance;
```

Comment:
I feel like we use previous instead of old here often for this kind of feedback loopy thing.